packages/terra-modal-manager/src/terra-dev-site/doc/modal-manager/About.1.doc.mdx

```diff
@@ -1,129 +1,71 @@
+import { Badge } from 'terra-modal-manager/package.json?dev-site-package';
+
+import ModalManagerExample from '../example/ModalManagerExample?dev-site-example';
+
+import ModalManagerPropsTable from '../../../../src/ModalManager?dev-site-props-table';
```

Comment:
All imports should be updated to pull from `terra-modal-manager`

Reply:
Done here a8a91ae